packages/terra-application/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-application",
+  "main": "lib/Application.scss",
+  "private": true,
+  "version": "0.3.0",
+  "description": "The application component sets global styles for the entire application. Global styles include font, colors, margins, and sizes.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-application",
+    "Application",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
```

Comment:
Also, since this doesn't have any js, do we want to remove these js dependencies?